ts/tiles-of-bbox.ts

```diff
@@ -15,6 +15,9 @@ export function getListOfTilesInBBox(bbox: IBBox, maxZ: number, minZ: number = 0
         const minY = lat2y(bbox.maxLat, z);
         for (let x = minX; x <= maxX; x += 1) {
             for (let y = minY; y <= maxY; y += 1) {
+                if (tms) {
```

Comment:
I would to prefer not to set another value to `y`. Can you remove the `if`-clause and change line 21  to:

```typescript
result.push(tms ? { z, x, Math.pow(2, z) - y - 1 } : { z, x, y });
```